app/src/views/GrantRegistryNewGrant.vue

```diff
@@ -1,80 +1,87 @@
-            description="The address contributions and matching funds are sent to"
-            id="payee-address"
-            label="Payee address"
-            :rules="isValidAddress"
-            errorMsg="Please enter a valid address"
-          />
-
-          <!-- Grant name -->
-          <BaseInput
-            v-model="form.name"
-            description="Your grant's name"
-            id="grant-name"
-            label="Grant name"
-            :rules="isDefined"
-            errorMsg="Please enter a name"
-          />
-
-          <!-- Grant name -->
-          <BaseInput
-            v-model="form.description"
-            description="Your grant's description"
-            id="grant-description"
-            label="Grant description"
-            :rules="isDefined"
-            errorMsg="Please enter a description"
-          />
-
-          <!-- Submit button -->
-          <button
-            type="submit"
-            class="btn btn-primary w-full text-center"
-            :class="{ 'btn-primary-disabled': !isFormValid }"
-            :disabled="!isFormValid"
-          >
-            Create Grant
-          </button>
-        </form>
-      </div>
+    <div class="text-left">
+      <form class="space-y-5" @submit.prevent="createGrant">
+        <p class="border-b border-grey-100"></p>
+
+        <!-- Grant name -->
+        <BaseInput
+          v-model="form.name"
+          placeholder="Fusion – Icon Pack for Cryptonauts"
+          id="grant-name"
+          label="Title"
+          :rules="isDefined"
+          errorMsg="Please enter a name"
+        />
+
+        <!-- Owner address -->
+        <BaseInput
+          v-model="form.owner"
+          placeholder="0xBADCdDEA250f1e317Ba59999232464933C4E8D90"
+          description="has permission to edit the grant"
+          id="owner-address"
+          label="Owner address"
+          :rules="isValidAddress"
+          errorMsg="Please enter a valid address"
+        />
+
+        <!-- Payee address -->
+        <BaseInput
+          v-model="form.payee"
+          placeholder="0xBADCdDEA250f1e317Ba59999232464933C4E8D90"
+          description="contributions and matching funds are sent to"
+          id="payee-address"
+          label="Payee address"
+          :rules="isValidAddress"
+          errorMsg="Please enter a valid address"
+        />
+
+        <!-- Grant Description -->
+        <BaseTextarea
+          v-model="form.description"
+          :placeholder="LOREM_IPSOM_TEXT"
+          id="grant-description"
+          label="Grant description"
+          :required="true"
+          :rules="isDefined"
+          errorMsg="Please enter a description"
+        />
+
+        <div class="grid grid-cols-12 border-b border-grey-100">
+          <div class="col-span-10 md:col-start-4 md:col-span-6 pt-5 pb-10 text-grey-400">
+            Your good to go deploy your grant on chain. You can add and edit your Grant’s detailed informations
+            afterwards.
```

Comment:
Seems this text is always there, even if the form is invalid? 

Also the phrasing feels weird since we aren't deploying a grant contract, and just submitting a transaction. I feel like we can just delete this whole `div` and change the button text to "Submit"